### PR TITLE
Initialize LOCAL file group

### PR DESCRIPTION
**Pull request by matthias-ronge**
The local filegroup was not initialized. I have no idea where that was lost because that was once there. I have now made a true constant of it, which is now possible.

### CODE REVIEW

File: Kitodo/src/main/java/org/kitodo/production/helper/metadata/legacytypeimplementations/LegacyInnerPhysicalDocStructHelper.java [hunk 1]
```diff
@@ -37,11 +37,15 @@
...
-     * A constant for local use. We cannot make this constant constant because
-     * the service loader is an instance variable.
+     * A constant for local use.
      */
     @Deprecated
-    public final MediaVariant local = new MediaVariant();
+    public static final MediaVariant LOCAL = new MediaVariant() {
+        {
+            setUse("LOCAL");
+            setMimeType("image/tiff");
```
Comment (henning-gerhardt) [inline Kitodo/src/main/java/org/kitodo/production/helper/metadata/legacytypeimplementations/LegacyInnerPhysicalDocStructHelper.java]: Should setting the mime type not depend of which kind of file is really used as local file? So far as I know institutes are using jpeg files for local usage and their tiff files are not used.

Comment (matthias-ronge) [reply Kitodo/src/main/java/org/kitodo/production/helper/metadata/legacytypeimplementations/LegacyInnerPhysicalDocStructHelper.java]: This is only a temporary solution as long as the helper classes for UGH replacement still exist. The value has always been written that way. It has never been used and read, it just needs to be in the field so there is no NullPointerException. Everything is configurable in the final solution. You do not even have to call the group LOCAL anymore.

**Comment by Kathrin-Huber** [comment]
> 
> 
> I have tests changes. It creates for me now mimetypes but when I want to open in editor changed process, editor crashes with unknown error. Debugging didn't show any error, but editor is not responsive anymore.
> 
> error

@matthias-ronge can you reproduce this?

**Comment by matthias-ronge** [comment]
No, it works for me. May you ask to @Beacze? We debugged this last week and it was some unrelated problem, but I don’t remember at the moment what it was.

**Comment by matthias-ronge** [comment]
Is there still something missing here? Or can you merge the branch now?

**Pull request merged**